### PR TITLE
Add java sdk example class

### PR DESCRIPTION
**Pull request by srcarrier**
This is a java class companion to the doc: https://cloud.ibm.com/apidocs/wh-acd?code=java

### CODE REVIEW

File: examples/AnnotatorForClinicalDataExample.java [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+/**
+ * Sample ACD Java SDK Calls
+ */
+public class AnnotatorForClinicalDataExample {
+
+	public static String ACD_APIKEY="<YOUR_APIKEY>";
+	public static String ACD_URL="https://us-east.wh-acd.cloud.ibm.com/wh-acd/api";
+	//public static String ACD_URL="https://us-south.wh-acd.cloud.ibm.com/wh-acd/api";
+	public static String version = "2020-06-01";
+	public static String name="AnnotatorForClinicalData";
```
Comment (dlangst) [inline examples/AnnotatorForClinicalDataExample.java:72]: Is 'name' variable at line 72 ever used?  At lines 68 and 69, should these be 'static final' (uppercase implies a constant)?

File: examples/AnnotatorForClinicalDataExample.java [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+		Response<ServiceStatus> resp = call.execute();
+		System.out.println("Response code: " + resp.getStatusCode() + " Status: " + resp.getStatusMessage());
+
+	}
+
+	// GET /cartridges
+	public static void getCartridges() {
+
+		CartridgesGetOptions opts = new CartridgesGetOptions();
+		/*
```
Comment (dlangst) [inline examples/AnnotatorForClinicalDataExample.java:432]: Why is this the code for this method commented out?

File: examples/AnnotatorForClinicalDataExample.java [hunk 3]
```diff
@@ -0,0 +1,543 @@
...
+		List<String> annotators = Arrays.asList(
+				Name.CONCEPT_DETECTION,
+				Name.NEGATION);
+
+		Flow flow = new FlowUtil.Builder()
+				.annotators(annotators).build();
+
+		String text = "Patient has lung cancer, but did not smoke. She may consider chemotherapy as part of a treatment plan.";
+
+		ContainerGroup response = acd.analyze(text, flow);
```
Comment (dlangst) [inline examples/AnnotatorForClinicalDataExample.java:137]: Should these examples demonstrate including a try-catch in the methods?  Currently only the 'deploy' methods show this?

File: examples/AnnotatorForClinicalDataExample.java [hunk 4]
```diff
@@ -0,0 +1,543 @@
...
+	public static void deleteProfile() {
+
+		DeleteProfileOptions options = new DeleteProfileOptions
+				.Builder()
+				.id("my_profile")
+				.build();
+
+		ServiceCall<Void> call = acd.deleteProfile(options);
+		Response<Void> resp = call.execute();
+		System.out.println("Response code: " + resp.getStatusCode());
```
Comment (dlangst) [inline examples/AnnotatorForClinicalDataExample.java:262]: For consistency, should we include a comment with the expected status code like we did in the previous two methods?

File: examples/AnnotatorForClinicalDataExample.java [hunk 5]
```diff
@@ -0,0 +1,543 @@
...
+			.Builder()
+			.id("my_flow")
+			.build();
+
+		ServiceCall<Void> call = acd.deleteFlows(opts);
+		Response<Void> resp = call.execute();
+		System.out.println("Response code: " + resp.getStatusCode());
+
+	}
+
```
Comment (dlangst) [inline examples/AnnotatorForClinicalDataExample.java:408]: There should a comment with 'Misc' header/separator here.

File: examples/AnnotatorForClinicalDataExample.java [hunk 6]
```diff
@@ -0,0 +1,543 @@
...
+import com.ibm.watson.health.acd.v1.model.ListStringWrapper;
+import com.ibm.watson.health.acd.v1.model.ServiceApiBean;
+import com.ibm.watson.health.acd.v1.model.ServiceStatus;
+import com.ibm.watson.health.acd.v1.model.UpdateFlowsOptions;
+import com.ibm.watson.health.acd.v1.model.UpdateProfileOptions;
+import com.ibm.watson.health.acd.v1.util.FlowUtil;
+
+/**
+ * Sample ACD Java SDK Calls
+ */
```
Comment (dlangst) [inline examples/AnnotatorForClinicalDataExample.java:65]: Do we want more verbiage here about the need to replace ACD_APIKEY, ACD_URL, and version variable values?

File: examples/AnnotatorForClinicalDataExample.java (hunk 2 above)
Comment (srcarrier) [reply examples/AnnotatorForClinicalDataExample.java:432]: The List<AcdCartridges> response from cartridgesGet(CartridgesGetOptions) wasn't merged into master at the time. The code has now been merged into master, but it has yet to be published in mvn central. So, the code will compile if you clone master locally, but not if you're consuming <= v1.1.0 of the sdk.

File: examples/AnnotatorForClinicalDataExample.java (hunk 3 above)
Comment (srcarrier) [reply examples/AnnotatorForClinicalDataExample.java:137]: I only added try/catches where exceptions were explicitly required to be handled.

**Pull request merged**